services/mailservers/fleet.go

```diff
@@ -0,0 +1,133 @@
+package mailservers
+
+func DefaultMailservers() []Mailserver {
+
+	return []Mailserver{
+		Mailserver{
+			ID:      "mail-01.ac-cn-hongkong-c.eth.prod",
+			Address: "enode://606ae04a71e5db868a722c77a21c8244ae38f1bd6e81687cc6cfe88a3063fa1c245692232f64f45bd5408fed5133eab8ed78049332b04f9c110eac7f71c1b429@47.75.247.214:443",
+			Fleet:   "eth.prod",
```

Comment:
Is it possible to use the consts from params/cluster.go instead of strings:
```
const (
	FleetUndefined  = ""
	FleetProd       = "eth.prod"
	FleetStaging    = "eth.staging"
	FleetTest       = "eth.test"
	FleetWakuV2Prod = "wakuv2.prod"
	FleetWakuV2Test = "wakuv2.test"
	FleetStatusTest = "status.test"
)
```